controls/cis_rhel8.yml

```diff
@@ -0,0 +1,2300 @@
+    automated: yes
+    rules:
+      - accounts_passwords_pam_faillock_deny
+      - accounts_passwords_pam_faillock_unlock_time
+      - var_accounts_passwords_pam_faillock_deny=5
+      - var_accounts_passwords_pam_faillock_unlock_time=900
+
+  - id: 5.4.3
+    title: Ensure password reuse is limited (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - accounts_password_pam_unix_remember
+      - var_password_pam_unix_remember=5
+
+  - id: 5.4.4
+    title: Ensure password hashing algorithm is SHA-512 (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - set_password_hashing_algorithm_systemauth
+
+  - id: 5.5.1.1
+    title: Ensure password expiration is 365 days or less (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - accounts_maximum_age_login_defs
+      - var_accounts_maximum_age_login_defs=365
+
+  - id: 5.5.1.2
+    title: Ensure minimum days between password changes is 7 or more (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - accounts_minimum_age_login_defs
+      - var_accounts_minimum_age_login_defs=7
+
+  - id: 5.5.1.3
+    title: Ensure password expiration warning days is 7 or more (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - accounts_password_warn_age_login_defs
+      - var_accounts_password_warn_age_login_defs=7
+
+  # TODO
+  # Rule doesn't check list of users
+  - id: 5.5.1.4
+    title: Ensure inactive password lock is 30 days or less (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - account_disable_post_pw_expiration
+      - var_account_disable_post_pw_expiration=30
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5537
+  - id: 5.5.1.5
+    title: Ensure all users last password change date is in the past (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 5.5.2
+    title: Ensure system accounts are secured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - no_shelllogin_for_systemaccounts
+
+  - id: 5.5.3
+    title: Ensure default user shell timeout is 900 seconds or less (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - accounts_tmout
+      - var_accounts_tmout=15_min
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5539
+  - id: 5.5.4
+    title: Ensure default group for the root account is GID 0 (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 5.5.5
+    title: Ensure default user umask is 027 or more restrictive (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
```

Comment:
How about this one as well?
accounts_umask_etc_login_defs

Reply:
Added